vnext/src/Libraries/Components/Keyboard/KeyboardExtProps.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+'use strict';
+
+import * as RN  from 'react-native';
+
+export enum EventPhase {
+  None = 0,
+  Capturing,
+  AtTarget,
+  Bubbling
+}
+
+export enum HandledEventPhase {
+  Capturing = 1, // Match with EventPhase
```

Comment:
Looking at the TypeScript [enum spec](https://www.typescriptlang.org/docs/handbook/enums.html) it seems like you might be able to have this literally be Capturing = EventPhase.Capturing.

Reply:
Very interesting assignment. will do

---
In reply to: [295561150](https://github.com/microsoft/react-native-windows/pull/2623#discussion_r295561150) [](ancestors = 295561150)